<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkSettingsDlg</class>
 <widget class="QDialog" name="NetworkSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#NetworkSettingsDlg
{   
color: transparent;
background-color: transparent;
border: 1px solid transparent;
border-radius: 8px; 
}

#DialogTitleBox
{   
color: black;
background-color: rgb(50,238,17);
border: 2px solid green;
border-top-left-radius: 8px; 
border-top-right-radius: 8px;
}
#DialogTitleLabel
{   
color: black;
}
#DialogItemsGroupBox
{   
background-color: rgb(50,238,17);
border: 2px solid green;
border-top-color: green; 
border-bottom-left-radius: 8px; 
border-bottom-right-radius: 8px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="m_TitleBarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>103</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="DialogItemsGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="m_StatusLabel">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_GoToAnchorSettingsButton">
        <property name="text">
         <string>Anchor Settings ( Select P2P Network )</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>This Device's Network Settings:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>How To Determine If Can Direct Connect (Is Port Open?)</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <item>
          <widget class="QRadioButton" name="AutoDetectProxyRadioButton">
           <property name="text">
            <string>Use Connection Test Node To Test For Firewall (recommended)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="AssumeNoProxyRadioButton">
           <property name="text">
            <string>Assume Users Can Connect Directly (No Firewall)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>External IP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_ExternIpEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="m_ExternIpHelpButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="AssumeProxyRadioButton">
           <property name="text">
            <string>Assume Relay Is Required (Device Is Behind Firewall).</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="WirelessIpLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="WirelessIpLabel">
          <property name="text">
           <string>Local Network Adapter IP ( recommend default )</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_LclIpListComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="m_UseUpnpCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enable UPNP to open router port (recommended)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="PortLabel">
          <property name="text">
           <string>Listen Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PortEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="RandomPortButton">
          <property name="text">
           <string>Ramdomize Port</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="m_IsPortOpenTesButton">
        <property name="text">
         <string>Test Is Port Open ( Can Direct Connect )</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
